src/sentry/integrations/bitbucket/integration.py

```diff
@@ -37,13 +38,57 @@ def get_client(self):
             self.model.external_id,
         )
 
+    def get_project_config(self):
+        client = self.get_client()
+        disabled = False
+        try:
+            repos = client.get_repos(self.model.name)
+        except ApiError:
+            # TODO(LB): Disable for now. Need to decide what to do with this in the future
+            # should a message be shown to the user?
+            # Should we try refreshing the token? For VSTS that often clears up the problem
+            repo_choices = []
+            disabled = True
+        else:
+            repo_choices = [(repo['uuid'], repo['full_name']) for repo in repos['values']]
+
+        try:
+            # TODO(LB): Will not work in the UI until the serliazer sends a `project_id` to get_installation()
+            # serializers and UI are being refactored and it's not worth trying to fix
+            # the old system. Revisit
+            default_repo = self.project_integration.config.get('default_repo')
```

Comment:
This suffers from the same issues that VSTS config does. Need to be sent a `project_id` from the serializer

Reply:
We fixed this in 7b441417c1 right?

Reply:
I wasn't aware of this pr. I'll look at it.